### ISSUE TITLE
Python linter flake8

### ISSUE THREAD

**Issue by RajatM24** [liquidev/lintplus#14]
i see no linter for python any way to build it plz guide or you can show me one

**Comment by RajatM24** [comment]
i mean it exists in the linter plugin on lint+

**Comment by liquidev** [comment]
The readme has a guide on creating linters: https://github.com/liquidev/lintplus#creating-new-linters

You can also take a look at existing linters to see how to create one that's compatible with lint+. Here's the luacheck linter, possibly the simplest one in the bunch:
https://github.com/liquidev/lintplus/blob/3813e3e985587086f35da9f449506a3c4d7e0696/lintplus_luacheck.lua#L12-L30

**Comment by RajatM24** [comment]
hey how do i get the patterns for warning, error, etc

**Comment by liquidev** [comment]
The patterns are standard [Lua patterns](https://www.lua.org/manual/5.2/manual.html#6.4.1) - these are very similar to regular expressions, but a lot simpler. I can show you how to write these if you give me some example output from the linter.

**Comment by RajatM24** [comment]
actually there are some patterns in the linter repository i will link both the repo and example

**Comment by liquidev** [comment]
That's exactly what I was asking for, thanks.

As far as I could tell from the docs, there are three main kinds of errors and warnings:
 - `E` are error codes
 - `F` and `W` are warning codes

To match errors, we would use the pattern:
```lua
"(.-):(%d+):(%d+): E%d+ (.+)"
```
The parentheses specify captures. We want to capture the filename, line, and column, and message, in this exact order.

Similarly, to match warnings, this pattern can be used:
```lua
"(.-):(%d+):(%d+): [FW]%d+ (.+)"
```
We use `[FW]` to match either one of the warning code types, F and W.

Note that I haven't tested these patterns as I'm currently at school, so they may need some tweaking in case I made an error.

You can refer to the Lua documentation for more info on what each individual part of the patterns means.

**Comment by RajatM24** [comment]
`-- mod-version:2
 local lintplus = require "plugins.lintplus" 
  
 lintplus.add("flake8") { 
   filename = "%.py$", 
   procedure = { 
     command = lintplus.args_command( 
       { "flake8", 
         lintplus.filename }, 
       "luacheck_args" 
     ), 
     interpreter = lintplus.interpreter { 
       warning = (.-):(%d+):(%d+): [FW]%d+ (.+)", 
       error = "(.-):(%d+):(%d+): E%d+ (.+)", 
     } 
   }, 
 } `
did i do it right
but it say no linter for the file

**Comment by RajatM24** [comment]
hey what about the asycn anyway is it working

**Comment by liquidev** [comment]
Currently async mode needs quite a bit of work to get working, but there's a section about it in the readme: https://github.com/liquidev/lintplus#asynchrony

There's an open issue about supporting Lite XL's built-in process API (#11), but I'm currently busy with other projects, so this issue has been laying dormant for quite a while.

**Comment by RajatM24** [comment]
looks like currently bat file does not exist
